spec/support/settings_helper.rb

```diff
@@ -1,11 +1,12 @@
-def stub_settings(hash)
+def stub_settings(hash, overwrite = false)
```

Comment:
arguably this would be better without the additional param `overwrite` and a new method e.g. `stub_settings_overwrite` would do this